.github/workflows/publish.yml

```diff
@@ -265,8 +265,6 @@ jobs:
         draft: true
         # APKs for webview-armeabi-v7a are not uploaded
```

Comment:
```suggestion
```

Now that Xwalk is gone, we have to publish APKs from the webview-armeabi-v7a architecture for old phones, like some Galaxy S.. and Motora G.. that has Android 5 or 6, and armeabi-v7a chipset.